build.gradle

```diff
@@ -1129,7 +1129,7 @@ publishing {
     repositories {
         maven {
             name = isRelease ? "SonaType" : "Artifactory"
-            url =  isRelease ? "https://oss.sonatype.org/service/local/staging/deploy/maven2/" : "https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot-local/"
+            url =  isRelease ? "https://oss.sonatype.org/service/local/staging/deploy/maven2/" : "https://us-central1-maven.pkg.dev/dsp-artifact-registry/libs-snapshot/"
             credentials {
                 username = isRelease ? project.findProperty("sonatypeUsername") : System.env.ARTIFACTORY_USERNAME
```

Comment:
This is where we authenticate to artifactory or sonatype.  Can you figure out what credentials we need for google and how to pass them?

Reply:
Same comment as here https://github.com/broadinstitute/gatk/pull/9198#discussion_r2136081920